python/requirements/ml/requirements_upstream.txt

```diff
@@ -4,6 +4,6 @@
 
 ray_lightning==0.3.0
 tune-sklearn==0.4.4
-xgboost_ray==0.1.10
-lightgbm_ray==0.1.5
-modin==0.12.1; python_version >= '3.7'
+xgboost_ray==0.1.15
+lightgbm_ray==0.1.8
+modin==0.18.1; python_version >= '3.8'
```

Comment:
we still need modin in the py 3.7 image for CI, right?

Reply:
I was considering removing it altogether...I don't think we need it at all. Users can install it themselves if they want.